pyfstat/make_sfts.py

```diff
@@ -92,31 +93,47 @@ def __init__(
         self.tbounds = [self.tstart, self.tend]
         logging.info("Using segment boundaries {}".format(self.tbounds))
 
-    def basic_setup(self):
+    def _get_sft_constraints_from_tstart_duration(self):
+        """
+        Use detectors, tstart and duration to set up a lalpulsar.SFTConstraints
+        object. This method is only used if noiseSFTs is not None.
+        """
+        SFTConstraint = lalpulsar.SFTConstraints()
+        SFTConstraint.detector = self.detectors
```

Comment:
Will this fail when `self.detectors` is a list? https://lscsoft.docs.ligo.org/lalsuite/lalpulsar/_s_f_tfile_i_o_8h_source.html#l00225 seems to only allow single-IFO constraints.